### PR TITLE
fix: updated the method of importing the parser

### PR DESCRIPTION
**Pull request by ayushnau**
Changes Added.
 
    1. updated the way the nunjucks filter was importing the generator. 

Related issue(s)

Related to: https://github.com/asyncapi/parser-js/issues/963

### CODE REVIEW

**Comment by ayushnau** [comment]
@smoya 
Fixed this.

image
image

In your example the error is coming due to asyncapi validation other wise the multiparser package is working fine. 

I have used this package for testing. @ayushnautiyal706020/multi-parser
Also pushed this code here if you want to check https://github.com/ayushnau/asyncapi-parser-example

**Comment by aeworxet** [comment]
@asyncapi/bounty_team

**Comment by smoya** [comment]
I don't get why we need this change. Can you please elaborate on that?

**Comment by smoya** [comment]
> In your example the error is coming due to asyncapi validation other wise the multiparser package is working fine.

Attaching two screenshots running a simple example both with `@smoya/multi-parser` package and then with `@asyncapi/multi-parser` one. See the error on the second.

#### Using `@smoya/multi-parser`
Code_0voe7Odx@2x

#### Using `@asyncapi/multi-parser`
Code_EsUr0mJD@2x

**Comment by ayushnau** [comment]
> I don't get why we need this change. Can you please elaborate on that?

Before I was using **parserv3: file::../parser** to import the parser in multiparser. but doing this way was creating problem in production build. of the multiparser. It turns out that in turborepo for importing internal dependency we need to use 
**@asyncapi/parser : "*"** this only works we are using the name which is the package.json of the package. So we need to use @asyncapi/parser instead of parserv3

**Comment by ayushnau** [comment]
> @ayushnau
> 
> > In your example the error is coming due to asyncapi validation other wise the multiparser package is working fine.
> 
> Attaching two screenshots running a simple example both with `@smoya/multi-parser` package and then with `@asyncapi/multi-parser` one. See the error on the second.
> #### Using `@smoya/multi-parser`
> 
> Code_0voe7Odx@2x
> #### Using `@asyncapi/multi-parser`
> 
> Code_EsUr0mJD@2x

yes I am aware of this. Merging this pr will fix this.

**Review by smoya — APPROVED** [review]

**Comment by smoya** [comment]
The prefix title of the pr should be fix.

File: .changeset/bubble-beach.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@asyncapi/multi-parser": patch
+---
+
+Adding the Way the generator is imported in Nunjucks filter.
```
Comment (smoya) [inline .changeset/bubble-beach.md:5]: Improve the message so it reflects what it fixes.

**Review by smoya — CHANGES_REQUESTED** [review]

File: .changeset/bubble-beach.md (hunk 1 above)
Comment (ayushnau) [reply .changeset/bubble-beach.md]: done.

**Review by smoya — APPROVED** [review]

**Pull request merged**